readme.md

```diff
@@ -98,22 +98,26 @@ Force enable or disable the `Inspect Element` menu item.
 Type: `boolean`<br>
 Default: `true`
 
-Show the `Look Up [selection]` menu item when right-clicking text on macOS.
+Show the `Look Up {selection}` menu item when right-clicking text on macOS.
 
 #### labels
 
 Type: `Object`<br>
 Default: `{}`
 
-Overwrite labels for the default menu items. Useful for i18n.
+Override labels for the default menu items. Useful for i18n.
+
+The placehlder `{selection}` may be used in any label, and will be replaced by the currently selected text.
+This is useful, for example, when localizing the `Look Up {selection}` menu item.
 
 Format:
 
 ```js
 {
 	labels: {
-		copy: 'Configured Copy',
-		saveImageAs: 'Configured Save Image As…'
+		copy: 'Copiar',
```

Comment:
Incorrect indentation here.

Reply:
Hmm, weird, in VSCode and SourceTree and `git diff` it looks fine.
It only looks wrong on the GitHub website with Refined Github enabled. If I disable Refined Github it looks fine here too.
It's definitely two tabs, same as before and same as the lines below.
Or am I missing something?

Reply:
Weird. Never mind.